### PR TITLE
Update Charts link, replace spaces with tabs in Make file

### PR DESCRIPTION
**Pull request by aamirqs**
Signed-off-by: Aamir <aameer.shaikh@mayadata.io>

### CODE REVIEW

**Review by prateekpandey14 — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -28,7 +28,7 @@ build:
 	@echo "------------------------------------"
      
 	helm init --client-only
```
Comment (prateekpandey14) [inline Makefile:30]: We can remove this now, helm 3 does not need this anymore

Comment (aamirqs) [reply Makefile:30]: For this we will have to update helm3 in the CI pipeline  as well. Can we let this merge go through and then then update helm operator in CI pipeline in another PR?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 The helm operator is typically built/updated upon one or both the below cases: 
 
-- New release of OpenEBS helm charts in [openebs/stable](https://artifacthub.io/packages/helm/openebs/openebs).
+- New release of OpenEBS helm charts in [openebs/stable](https://github.com/openebs/charts).
```
Comment (kmova) [inline CONTRIBUTING.md:5]: n_: change this to openebs/charts?

Comment (kmova) [inline CONTRIBUTING.md:5]:
```suggestion
- New release of OpenEBS helm chart is available in artifacthub.io [openebs/openebs](https://artifacthub.io/packages/helm/openebs/openebs).
```

**Review by kmova — APPROVED** [review]

**Pull request merged**